DigitalLearningSolutions.Web/Views/SuperAdmin/Centres/_CentreCard.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Centres
+@model CentreSummaryViewModel
+
+<div class="searchable-element nhsuk-expander-group" id="@Model.CentreId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.CentreId-name" name="centre-name">
+        @Model.CentreName (@Model.CentreId)
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <dl class="nhsuk-summary-list">
+
+        <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre id
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreId
+        </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Centre type
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.CentreType
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Region
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.RegionName
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Contact
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.ContactForename @Model.ContactSurname
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Contact email
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.ContactEmail
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Telephone
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.ContactTelephone
+      </dd>
+    </div>
+
+    </dl>
+    <a class="nhsuk-button nhsuk-u-margin-right-2 nhsuk-u-margin-bottom-2"
+       role="button"
+       href="#">
+      Manage centre
+    </a>
+    <a class="nhsuk-button nhsuk-u-margin-right-2 nhsuk-u-margin-bottom-2"
+       role="button"
+       href="#">
+      View courses
+    </a>
+    @if (Model.Active)

```

Comment:
I noticed there was a question on the ticket description about a potential "reactivate button". Was there a decision made about this?

Reply:
It is still outstanding, Carolyn is away right now. I have planned in a more detailed discussion with Carolyn, not just on this but the whole Super Admin section.

Reply:
Good to know. Seeing as it will be a relatively simple change (if required), and everything else in the review seems good to me, I'm happy to move this on and the change can be tackled during tech lead review (or in another ticket if the decision is made far in the future for some reason)

Reply:
I've now added the button following confirmation.